data/hai missions.txt

```diff
@@ -1053,3 +1053,251 @@ mission "Hai Honeymoon"
+			choice
+				`(Continue.)`
+			`Chaos spreads amongst the other captains, and the group sitting with you is in anguish. Xavier collapses back to the table, holding his head in both hands, "Those paintings and sculptures were our main attraction! They're worth millions! It'll take a month before the Navy ever notices the pirates there and make it safe for the convoy to come here. The art exhibition is scheduled two weeks from now!" As they try to figure out what to do, Nanachi speaks up,"I'll go there, I pilot a human ship and if we just need to transfer cargo from one ship to the other, I could get out of there quickly, nobody would notice that a Hai ever went there."`
+			`"Are you insane? If they managed to take down that Leviathan, you wouldn't last a second! Besides that convoy was extremely fast, your chances of outrunning them are next to nothing!" Vanessa says. "Captain <last>," Nanachi turns to you, "I know you already helped me way more than anyone would ever be willing to, and I'll never forget that, but would you be willing to help me reach that planet, and get back here?" While it's certain that for a pirate fleet to take down a Leviathan so easily they must be incredibly powerful, and this could be extremely dangerous, you're sure that if Nanachi tries to get to Prime by herself she'll just get herself killed.`
+			choice
+				`"Alright, I'll make sure we head to Prime and get back safely."`
+				`"No, I agreed to help you find and deal with some simple jobs, I can't risk fighting such a fleet by myself."`
+					decline
+			`She thanks you profusely, and promises that she'll pay you as soon as she can. The organizers are a bit hesitant, but seeing as no other captain here decided to offer them help, they agree to let you two go to Prime retrieve the cargo. As you're leaving the bar, Xavier pulls you aside, and whispers, "<last>, I know you're just trying to help that girl, but don't get killed over this, if you run into these pirates don't try to fight them, just rush to Prime."`
+				accept
+	npc
+		personality nemesis waiting
+		government "Pirate"
+		system "Betelgeuse"
+		ship "Marauder Firebird (Hai)" "Nut King Call"
+	npc
+		personality nemesis waiting
+		government "Pirate"
+		system "Betelgeuse"
+		ship "Marauder Splinter (Hai)" "The Nutcracker"
+	npc
+		personality nemesis waiting
+		government "Pirate"
+		system "Betelgeuse"
+		ship "Marauder Quicksilver (Hai)" "Hazelnut's Bane"
+	npc
+		personality nemesis waiting
+		government "Pirate"
+		system "Betelgeuse"
+		ship "Marauder Bounder (Hai)" "Rodent Hunter"
```

Comment:
Do these all need to be separate `npc`s? Can you not just put them all under one block?